lib/conjur/api/roles.rb

```diff
@@ -19,9 +19,32 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 require 'conjur/role'
+require 'conjur/graph'
 
 module Conjur
   class API
+    ##
+    # Fetch a digraph (or a forest of digraphs) representing of
+    #   role memberships related transitively to any of a list of roles.
+    #
+    # @param [Array<Conjur::Role, String>] roles the  digraph (or forest thereof) of
+    #   the ancestors and descendants of these roles or role ids will be returned
```

Comment:
We allowed user to see (_transitively_) what roles belong to his. I'm not sure we want that anymore. (That'd be the lower cone on the image I drew the other day.)

Eg. do we want the user to be able to see members of admin if admin is a member of the user?